docs/modules/about/pages/welcome.adoc

```diff
@@ -1,54 +1,40 @@
+First, your provisioned network inventory is analyzed and converted to a graphical representation.
+Inventory objects that are analyzed include nodes, their components, and their relations to other nodes.
 
-=== We group alarms on the graph into clusters
+.Example network inventory graph
+image::model.png[Network inventory graph that displays ten inventory objects and their directional relations, 600]
 
-We aim to group the alarms into clusters based on their level or similarity, and whether or not they share the same root cause.
-The engine will periodically group (cluster) the alarms.
+After the graph is generated, alarms are enriched with additional metadata: the managed object type, and the alarm's managed object instance.
+This metadata is required to identify which components the alarms are related to.
 
-image::model_with_alarms_in_cluster.png[Model example with cluster,600]
+Alarms are then attached to the graph as they are triggered, and grouped into clusters based on their level of similarity and whether they share the same root cause.
+When ALEC determines that a cluster of alarms is related, an event is sent to OpenNMS.
+OpenNMS then creates a new situation for the cluster.
 
-Different clustering algorithms can be used:
+.Example network inventory graph with alarm cluster and situation
+image::model_with_situation.png[Network inventory graph that displays ten inventory objects, their directional relations, a cluster of four attached alarms, and a situation, 600]
 
-1. Deep Learning (AI with TensorFlow) 
-2. DB-Scan (Unsupervised ML)
+ALEC uses https://www.drools.org/[Drools] rules to manage the state of all alarms, including situations:
 
-=== We create situations for related alarms
+* The severity of a situation is the maximum severity of all related alarms, plus one.
+* If all alarms related to a situation are acknowledged, the situation is also acknowledged.
 
-When we determine that a group of alarms is related, we send an event to OpenNMS.
-OpenNMS will then create a new *situation* for the set of related alarms.
-Situations are also managed like alarms.
+== Correlation engines
 
-image::model_with_situation.png[Model example with situation,600]
+ALEC implements two correlation engines:
 
-We use Drools rules for managing the state of all alarms including situations:
+* *Deep Learning:* an algorithm that leverages deep learning techniques through a neural network to correlate and group alarms.
+* *DBSCAN:* leverages the https://en.wikipedia.org/wiki/DBSCAN[DBSCAN algorithm] to group alarms based on their distance from each other.
 
-* These are used to propagate the severity:
-** the severity of a situation is the maximum severity of all related alarms + 1
-* They are used to propagate acknowledgments:
-** if all of the related alarms on a situation are acknowledged, then the situation is also acknowledged
+To learn more, see xref:engines:deeplearning.adoc[Deep Learning] and xref:engines:dbscan.adoc[DBSCAN].
```

Comment:
These are not showing up as links, but that could just be a build thing.

Reply:
Could be... they show up as links in the VSC preview, so it should be okay. 🤔